packages/kbn-securitysolution-io-ts-types/src/time_duration/index.ts

```diff
@@ -26,7 +26,7 @@ export const TimeDuration = new t.Type<string, string, unknown>(
         if (
           time >= 1 &&
           Number.isSafeInteger(time) &&
-          (unit === 's' || unit === 'm' || unit === 'h')
+          (unit === 's' || unit === 'm' || unit === 'h' || unit === 'd')
```

Comment:
What was the reason for this change?

Reply:
If the throttle payload is 'Weekly' the value is `7d`, and since throttle now depends on `TimeDuration` I had to extend it